js/default.js

```diff
@@ -81,6 +88,16 @@ var terminal = {
                     return callback(data);
                 });
             },
+            monitor: function(command) {
+                var every = "10";
```

Comment:
why `"10"`? I'm not sure what this does. `null` it up and check for `null` values